### ISSUE TITLE
FedEx RESTFul API

### ISSUE THREAD

**Issue by mrmaloof** [pullingshots/Shipment#48]
Our customers are being contacted by FedEx reps and being encouraged to upgrade their FedEx integrations to the FedEx RESTful API. Are there any plans to use the FedEx RESTFul API?

**Comment by pullingshots** [comment]
Yes, I can absolutely work on this. Have you heard any indication of when they are retiring the SOAP/XML interface?

**Comment by mrmaloof** [comment]
I'm not sure of the timeline but here is a message one of our customers forwarded to us...

I am with the API Core Team at FedEx Services and I am hoping to get your assistance on finding out who I need to get with to get one of your older software Web Service meters migrated to our newer RESTful API. The meter your currently using is very old and we would like to see if it would be feasible to begin working towards the RESTful API upgrade. The new RESTful API has been available to customers for the past year or so and are going to be moving forward with that model for all customers. We have put the Web Services into software containment with no more new development will be done on that and we are starting to migrate customers from the Web Services platform to the new RESTful API platform. The thought was to start with customers using the oldest versions and move forward. So if you could steer me in the right direction as far as who to talk with regarding this, that would be great.

**Comment by icyavocado** [comment]
I will take a look here.
